templates/eng-workflow/hgpush.1.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "hgpush",
+  "properties": {
+    "changesetID": {
+      "description": "The 40 char hex ID of the changeset. See https://www.mercurial-scm.org/wiki/ChangeSetID",
+      "type": "string",
+      "pattern": "[a-f0-9]{40}"
+    },
+    "reviewSystemUsed": {
+      "description": "Which code review system was used to review this changeset?",
+      "type": "string",
+      "enum": ["bmo", "mozreview", "phabricator", "unknown", "not_applicable"]
```

Comment:
We've been leaning away from overly-strict schemas like this, on the off-chance that you update the client with a new review system and forget to add it to the enum here. I would say a string should suffice.